main.js

```diff
@@ -1,7 +1,10 @@
 (function(){
   var node = document.querySelector("div");
   var app = Elm.Main.embed(node, {
-      eventId: window.location.href.split('/events/')[1] ? window.location.href.split('/events/')[1].split('/')[0] : ""
+      eventId: window.location.href.split('/events/')[1] ? window.location.href.split('/events/')[1].split('/')[0] : "",
+      host: window.location.origin,
+      showPreviewUi: window.location.href.split('/programme-builder')[1] === '/preview',
```

Comment:
this is tightly coupling this sub app with our main app. We should pass this data to the sub app, rather than have it be dependant upon having the corrrect URL